omod/src/main/web/dashboardwidgets/dataintegrityviolations/dataintegrityviolations.controller.js

```diff
@@ -18,7 +19,12 @@ export default class DataIntegrityViolationsController {
             this.config.maxResults = 6;
         }
 
-        this.openmrsRest.list('dataintegrity/integrityresults', { patient: this.config.patientUuid, v: 'full', limit: this.config.maxResults }).then((resp) => {
+        this.openmrsRest.list('dataintegrity/integrityresults', {
```

Comment:
@shivtej1505 Do not make changes for the sake - leave it that way, make it work then come back and cleanup. That change does not add any value today - but could be causing some issues elsewhere